### PR TITLE
feat(universal-express): adding schematic to run universal schematic and then create express server

### PR DESCRIPTION
**Pull request by edoparearyee**

### CODE REVIEW

File: src/testing/index.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import * as data from './data';
 import { TestingSchema } from './schema.model';
 
 export default function(options: TestingSchema): Rule {
-  console.log(JSON.stringify(options, null, 2));
+  // console.log(JSON.stringify(options, null, 2));
```
Comment (jackhkmatthews) [inline src/testing/index.ts]: Do you still want this console log here?

File: src/universal-express/files/server.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+// Faster server renders w/ Prod mode (dev mode never needed)
+enableProdMode();
+
+// Express server
+const app = express();
+
+const port = process.env.PORT || 4000;
+const distFolder = join(process.cwd(), 'dist');
+
+// Our index.html we'll use as our template
```
Comment (jackhkmatthews) [inline src/universal-express/files/server.ts]: Not sure what this comment means

File: src/universal-express/index.ts [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+} from '@angular-devkit/schematics';
+import { strings } from '@angular-devkit/core';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { Schema as UniversalSchema } from '@schematics/angular/universal/schema';
+import { addImportToModule } from '@schematics/angular/utility/ast-utils';
+import { InsertChange } from '@schematics/angular/utility/change';
+import * as ts from 'typescript';
+
+export default function(options: UniversalSchema): Rule {
+  // console.log(JSON.stringify(options, null, 2));
```
Comment (jackhkmatthews) [inline src/universal-express/index.ts]: do you still want this console log here?

File: src/universal-express/index_spec.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const collectionPath = path.join(__dirname, '../collection.json');
+
+
+describe('sn', () => {
+  it('works', () => {
+    const runner = new SchematicTestRunner('schematics', collectionPath);
+    const tree = runner.runSchematic('my-schematic', {}, Tree.empty());
+
+    expect(tree.files).toEqual([]);
+  });
```
Comment (jamesjwarren) [inline src/universal-express/index_spec.ts]: Is it worth adding any other tests, like comparing the output to an example generated file?

**Review by jackhkmatthews — CHANGES_REQUESTED** [review]
Just the TS loader version as discussed.

File: src/universal-express/files/server.ts (hunk 2 above)
Comment (edoparearyee) [reply src/universal-express/files/server.ts]: remove the variable as it wasn't used anyway

File: src/universal-express/index.ts (hunk 3 above)
Comment (edoparearyee) [reply src/universal-express/index.ts]: removed

File: src/universal-express/index_spec.ts (hunk 4 above)
Comment (edoparearyee) [reply src/universal-express/index_spec.ts]: @jamesjwarren tests have been added: https://circleci.com/gh/thisissoon/schematics/2

**Comment by edoparearyee** [comment]
@jamesjwarren / @jackhkmatthews should be ready for another review now

**Pull request merged**